src/expressionEvaluator/iExpressionEvaluator.ts

```diff
@@ -0,0 +1,5 @@
+import { JSExpr } from './jsExpr'
+
+export interface IExpressionEvaluator {
```

Comment:
let's not use I prefix.. difficult to remember what is an Interface, an enum, a type...

Reply:
you are right, but I didn't know how to call it... I could not pass the ExpressionEvaluator itself to avoid circular dependencies in the imports. How can we call it?!